src/providers/MeetingProvider/index.tsx

```diff
@@ -46,31 +51,47 @@ export const MeetingProvider: React.FC<Props> = ({
   logger,
   videoDownlinkBandwidthPolicy,
   meetingManager: meetingManagerProp,
+  enableMeetingEvents = false,
   children,
 }) => {
   const [meetingManager] = useState(
     () => meetingManagerProp || new MeetingManager({ logLevel, postLogConfig, simulcastEnabled, logger, videoDownlinkBandwidthPolicy })
   );
 
+  const wrapProviders = (children: ReactNode) => 
+    <AudioVideoProvider>
+      <DevicesProvider>
+        <RosterProvider>
+          <RemoteVideoTileProvider>
+            <LocalVideoProvider>
+              <LocalAudioOutputProvider>
+                <ContentShareProvider>
+                  <FeaturedVideoTileProvider>
+                    {children}
+                  </FeaturedVideoTileProvider>
+                </ContentShareProvider>
+              </LocalAudioOutputProvider>
+            </LocalVideoProvider>
+          </RemoteVideoTileProvider>
+        </RosterProvider>
+      </DevicesProvider>
+    </AudioVideoProvider>;
+
+  const wrapMeetingEventProvider = (children: ReactNode) => {
+    const wrappedProviders = wrapProviders(children);
+    if (enableMeetingEvents) {
+      return (
+        <MeetingEventProvider>
+          {wrappedProviders}
+        </MeetingEventProvider>
+      );
+    }
```

Comment:
Is there any downside for just including the MeetingEventProvider regardless of if the enableMeetingEvents is enabled or not? Just like all of the Providers above - we don't conditionally include them under MeetingProvider, we just include them by default. The user can just choose if they want to use the Hook or not, right?